src/components/Popup.js

```diff
@@ -20,6 +20,16 @@ export default class Popup {
     document.removeEventListener("keydown", this._handleEscClose);
   }
 
+  toggleUploadIndicator(isUploading) {
```

Comment:
`toggleUploadIndicator` should not be in `Popup` because the image modal doesn't have a submit button. Please, move it into `PopupWithForm`